### ISSUE TITLE
Convert security-secretstore-setup to use common bootstrap package

### ISSUE THREAD

**Issue by michaelestrin** [edgexfoundry/edgex-go#1806]
See #1739 for implementation and example conversion of core-data.

Add @michaelestrin and @tsconn23 as reviewers to your pull request.

Address any questions on implementation to @michaelestrin.

**Comment by michaelestrin** [comment]
@tingyuz:

1. Should security-secretstore-setup have the option to connect to the registry to retrieve it's configuration?

1. How does one test this application to ensure it's functioning properly?

If you're not the appropriate person to answer these questions, please @ the appropriate person here.

**Comment by michaelestrin** [comment]
Existing PR #1793 (issue #1677) converts security-secretstore-setup to use common bootstrap package.

**Comment by tingyuz** [comment]
> @tingyuz:
> 
> 1. Should security-secretstore-setup have the option to connect to the registry to retrieve it's configuration?
> 2. How does one test this application to ensure it's functioning properly?
> 
> If you're not the appropriate person to answer these questions, please @ the appropriate person here.

1. Yes it was discussed and decided that security-secretstore-setup needs to support such feature.
2. As security-secretstore-setup is still being developed actively, for testing one of the options is to insert logs that print out all the configuration settings which are retrieved from registry, this way we will be able to confirm all the required settings for the component are available when they are needed. Once the PR for the features within secretstore-setup is merged we can run the test again to make sure it works as expected.

**Comment by michaelestrin** [comment]
In conversation with @tsconn23, @tingyuz, and @AnthonyMBonafide today, agreed to put this issue on hold until after October Fuji freeze.  Follow-on Fuji security release will address this issue.

**Comment by bnevis-i** [comment]
This migrating to common bootstrap is permanent fix for #1814.

**Comment by michaelestrin** [comment]
Abandoned prior implementation.
